Documents/Institutions.md

```diff
@@ -0,0 +1,62 @@
+# Institutions Data Model
+
+This document describes the data model for Financial Institutions in HMDA. 
+A Financial Institution Database is created in HMDA for the purposes of 
+filing mortgage data. The information in this database is used in some edits, 
+as complementary information for publication and analysis of mortgage data, 
+and in the creation of the HMDA Panel. The HMDA Panel is the list of institutions
+that are required to file HMDA data, every year. 
+
+The Institutions Database needs to keep historical information for analysis, 
+and at least have information readily available for the two previous years in order
+to be able to perform the necessary checks when performing validation on the HMDA
+data being submitted or when composing the HDMA Panel. In addition to this, it must
+maintain the relationships between institutions (i.e. parent/child relationships)
+ and the actions that modify them (i.e. mergers, acquisitions). 
+ 
+ 
+The end product is a list of institutions that contains the necessary information 
+so that the HMDA Panel can be processed automatically. This table contains the following
+information:
+
+
+* `Activity Year` (integer length 4, i.e. 2017)
```

Comment:
Is activity year more of an indicator that the institution sent a HMDA filing for a particular year? Are we envisioning one row for each institution per year? It makes sense to track history of which years institutions file.

Do we need a date for when this information is as of? For instance, we needs assets as of 12/31 of the prior year (so 12/31/2016 for 2017 data). Even if we get an update to assets for an institution on say a refresh on 4/30/2017, we do not want to update the assets for an institution because all that matters is what they were on 12/31 of the prior year. The same goes for MSA - it is based on which branches were in existence on 12/31 of the prior year. Just something to consider if we end up updating data monthly or quarterly that we will need to indicate as of what date the information contained on each record is.